app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -181,7 +181,7 @@ def send_user_otp(method)
       params = {
         to: phone_to_deliver_to,
         otp: current_user.direct_otp,
-        expiration: Devise.direct_otp_valid_for.to_i / 60,
+        expiration: TwoFactorAuthenticatable.direct_otp_valid_for_seconds / 60,
```

Comment:
what if we defined a `_minutes`/`_MINUTES` version of this method/constant?

Reply:
added in 0e0a3a6ad